### ISSUE TITLE
Share via email had problems

### ISSUE THREAD

**Issue by jeeznews** [BusinessUSA/BusinessUSA-Usability-Feedback#36]
I was able to share via email, but the window that opened was incredibly tiny and difficult to navigate. Even the "send" button was small. After I sent it, I could not navigate back to the home page.

**Comment by MagsStew** [comment]
I agree. If the user opens the email selection but then decides to send via other method - Linked In, etc., they could become stuck there because they do not realize they can click on the 'black area" to see the page their results are on. 
![3_2015_share](https://cloud.githubusercontent.com/assets/8241312/6530529/9df3a434-c3ff-11e4-89e3-474ac4a484c3.jpg)
